### ISSUE TITLE
C API documentation index is laid out incorrectly

### ISSUE THREAD

**Issue by iritkatriel** [python/cpython#105196]
The last two lines in this snippet should be indented further than the line above them, they are more specific.
(Alternatively, they can be removed because they don't add much).

<img width="401" alt="image" src="https://github.com/python/cpython/assets/1055913/5bee1ab5-4a6f-45a7-b0fb-de2710002c10">

### Linked PRs
* gh-105672
* gh-105782
* gh-105786

**Comment by rhettinger** [comment]
I would just fix the indentation on the last two.  Then, I would check to make sure the link targets are unchanged so that external links into our docs aren't broken.

**Comment by iritkatriel** [comment]
I think Ellipsis should be indented under slice as well:

<img width="194" alt="image" src="https://github.com/python/cpython/assets/1055913/ca8f0f8d-f323-4b97-b93c-47989cf787bc">

**Comment by iritkatriel** [comment]
An internal frames under frame objects:

<img width="225" alt="image" src="https://github.com/python/cpython/assets/1055913/1e047383-34fd-4c68-8049-d8ca0ddf3585">

**Comment by safwansamsudeen** [comment]
Hey, I'm new here, would love to contribute to Python!

It's not much of a fix, but can I work on this?

**Comment by roy9495** [comment]
Hi, I made the indentation changes in these two files Doc/build/html/contents.html, Doc/build/html/c-api/concrete.html respectively but these files are untracked so when I am trying to commit the commit is becoming too large. 
Any guidance you can give me will be helpful.

**Comment by safwansamsudeen** [comment]
@roy9495, were you assigned to this?

**Comment by roy9495** [comment]
@safwansamsudeen No, I was not assigned. Sorry, I should have asked first.

**Comment by iritkatriel** [comment]
We don’t assign issues to contributors. 

The way it goes is that if you want to work on an issue you leave a comment about that, and if there is such a comment by someone else, and it’s a fairly recent comment, you let them work on it. If it’s an older comment you should ask them if they are still working on it, and if they don’t reply within a reasonable amount of time you can leave a comment to indicate that you are working on it.

So basically, we communicate our intentions on the issue.

**Comment by roy9495** [comment]
@safwansamsudeen Are you still working on this?

**Comment by safwansamsudeen** [comment]
@roy9495 I didn't start it. If you're working on it, that's cool! If you're busy, I'll take it up.

**Comment by safwansamsudeen** [comment]
> @safwansamsudeen No, I was not assigned. Sorry, I should have asked first.

Ahh Roy I'm sorry, I misread the conversation. My first comment on this issue indicated that I wanted to work on this, and I interpreted your comment to be disregarding mine and working on it instead... which is why I asked if you were assigned to it.

I now see you were responding to my request of help, ugh. Thanks, and do accept my apologies for my coldness in my second comment! And great work on the PR!

**Comment by roy9495** [comment]
@safwansamsudeen Thank you, Please don't apologise.

**Comment by CAM-Gerlach** [comment]
PR and all backports merged, so closing this now. Thanks @roy9495 and @iritkatriel!
